pkgs/top-level/all-packages.nix

```diff
@@ -5592,6 +5592,8 @@ with pkgs;
 
   hyprshade = python311Packages.callPackage ../applications/window-managers/hyprwm/hyprshade { };
 
+  hyprnotify = callPackage ../by-name/hy/hyprnotify/package.nix { };
```

Comment:
```suggestion
```

Reply:
Not sure what you mean. What I'm saying is to remove the entry in all-packages.nix that you added

Reply:
I was unaware that change had been made... Interesting...